dvl-a50/mavlink2resthelper.py

```diff
@@ -166,24 +169,29 @@ def __init__(self):
 }}
 """
 
-    def get_float(self, path: str) -> float:
+    def get_float(self, path: str, vehicle: int=None, component: int=None) -> float:
```

Comment:
Per @patrickelectric's comments on #6, the types here should be `Optional[int]`
```suggestion
    def get_float(self, path: str, vehicle: Optional[int]=None, component: Optional[int]=None) -> float:
```
(probably requires importing `Optional` from `typing`)